docs/getting-started/install-zowe-sdks.md

```diff
@@ -0,0 +1,87 @@
+# Zowe SDKs installation
+
+Leverage the Zowe Client Software Development Kits (SDKs) to build client applications and scripts that interface with the mainframe.
+
+The SDKs include programmatic APIs, each of which performs a particular mainframe task. For example, one API package provides the ability to upload and download z/OS data sets. You can leverage that package to rapidly build a client application that interacts with data sets.
+
+The following SDKs are available.
+- Zowe Client Node.js SDK
+- Zowe Client Python SDK *technical preview*
+- Zowe Client Kotlin SDK *technical preview*
+- Zowe Client Java SDK *under development*
+
+## SDK documentation
+
+For detailed SDK documentation, see the following:
+- [Zowe Node.js SDK](https://docs.zowe.org/stable/typedoc/index.html)
+- [Zowe Client Python SDK](https://zowe-client-python-sdk.readthedocs.io/en/latest/)
+
+## Software requirements
+
+### Node.js
+
+If you install Node SDK packages from the online registry, the required dependencies are installed automatically.
+
+If you download Node SDK packages from Zowe.org, the folder contains dependencies that you must install manually. Extract the `TGZ` files from the folder, copy the files to your project, and issue the following commands to install the dependencies.
+
+```
+npm install imperative.tgz
+```
+
+```
+npm install core-for-zowe-sdk.tgz
+```
+
```

Comment:
Can you add a java section here. For instance:

### Java

Requires Java runtime version 11 and above.

Reply:
Hey @frankgiordano,
I like the idea of adding a Java section here. 😋 

Curious if we should align with the server's minimum Java version (which I believe is 17 for Zowe V3)

Reply:
Since this content is slated for the Zowe V3 release, I am adding "Requires Java runtime version 17 and above."

Reply:
@frankgiordano @zFernand0 -- what would this Java requirement apply to? This is a list of software requirements that is broken down by SDK: Node.js, Python, Kotlin, and Java. Where would the requirement statement "Requires Java runtime version 11 and above." go? Is this specific to only the Java sdk? Or all sdks?
<img width="482" alt="image" src="https://github.com/zowe/docs-site/assets/109533923/1603b86e-39c9-4939-9bc9-f7c62813ed8a">

Reply:
@frankgiordano, @KUGDev,
Please correct me if I'm wrong, but the same Java Runtime Environment (JRE) is required for both SDKs 😋